docs/tutorials/new_dataset.md

```diff
@@ -278,6 +287,27 @@ There two ways to concatenate the dataset.
         test = dataset_A_test
         )
     ```
+    If the concatenated dataset is used for test or evaluation, this manner also supports to evaluate each dataset separately.
+
+3. We also support to define `ConcatDataset` explicitly as the following.
+
+    ```python
+    dataset_A_val = dict()
+    dataset_B_val = dict()
+
+    data = dict(
+        imgs_per_gpu=2,
+        workers_per_gpu=2,
+        train=dataset_A_train,
+        val=dict(
+            type='ConcatDataset',
+            datasets=[dataset_A_val, dataset_B_val],
+            separate_eval=False))
+    ```
+    This manner allows users to evaluate all the datasets as a single one by setting `separate_eval=False`.
+
+**Note:**
+This option `separate_eval=False` assumes the dataset uses `self.data_infos` during evaluation. Therefore, COCO datasets does notsupport this behavior since COCO datasets do not fully rely on `self.data_infos` during evaluation. Combining different type ofdatasets and evaluating them as a whole is not tested thus is not suggested.
```

Comment:
Check grammars, e.g. `Coco datasets does notsupport` --> `Coco dataset does not support`.